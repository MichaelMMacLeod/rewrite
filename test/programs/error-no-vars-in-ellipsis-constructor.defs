(def pattern
  ((no vars here) ..))