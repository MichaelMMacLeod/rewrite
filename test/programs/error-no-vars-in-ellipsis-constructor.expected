error[E008]: no variables in term preceding ellipsis
./test/programs/error-no-vars-in-ellipsis-constructor.defs:2:19
   |
 2 |   ((no vars here) ..))
   |                   ^^ the ellipsis
./test/programs/error-no-vars-in-ellipsis-constructor.defs:2:4
   |
 2 |   ((no vars here) ..))
   |    ^^^^^^^^^^^^^^ the term preceding the ellipsis
help: there must be at least one variable in the term preceding an ellipsis
